### ISSUE TITLE
mrcfile file corruption warnings (APPLE)

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#428]
In the process of processing some data (10025 20S) with APPLE picker the following warnings are generated.  The output of APPLE seems reasonable so far; I think the warnings are benign.

I mentioned this to Joakim and he suggested there may be a more permissive mode switch for the `mrcfile` package.  Will try that.  If that doesn't work we may instead want to capture this warning and log our own (preferable just once).

```
/home/garrett/anaconda3/envs/aspire_APPLE/lib/python3.6/site-packages/mrcfile/mrcinterpreter.py:209: RuntimeWarning: Map ID string not found - not an MRC file, or file is corrupt
  warnings.warn(msg, RuntimeWarning)
/home/garrett/anaconda3/envs/aspire_APPLE/lib/python3.6/site-packages/mrcfile/mrcinterpreter.py:219: RuntimeWarning: Unrecognised machine stamp: 0x00 0x00 0x00 0x00
  warnings.warn(str(err), RuntimeWarning)
```

**Comment by garrettwrong** [comment]
Looking into this I see we are already attempting to open the file permissively.  This is a case of a corrupt data fileset (every single file has a bad map and bad machine stamp).  I will try to emit a more meaningful message in ASPIRE and log the issues with the file to debug logs.

**Comment by garrettwrong** [comment]
Closed by #429
